src/App.jsx

```diff
@@ -16,10 +18,15 @@ function App() {
     dispatch(fetchMissions());
   }, [dispatch]);
 
+  useEffect(() => {
+    dispatch(getRockets());
+  }, [dispatch]);
+
   return (
     <Routes>
       <Route path="/" element={<Layout />}>
         <Route index element={<Missions />} />
+        <Route path="/rockets" element={<Rockets />} />
```

Comment:
I think you should replace the Missions component and link with the Rockest component and link as the home page in order to align with the setup of the medium-fidelity wireframes.

- The medium-fidelity wireframe links
![links](https://user-images.githubusercontent.com/90119799/223978759-f2e6e026-5ece-45f2-80ce-102a18522d6e.png)

- Our page
![Shakir links](https://user-images.githubusercontent.com/90119799/223978922-29db18d7-e802-4414-9e77-1583808a4b70.png)

Reply:
Thanks, @Timbar09, for the suggestion. I will make these changes ASAP.